cekit/generator/base.py

```diff
@@ -126,6 +127,32 @@ def override(self, overrides_path):
         descriptor.merge(self.image)
         return descriptor
 
+    def _generate_expose_services(self):
+        """Generate the label io.openshift.expose-services based on the port
+        definitions."""
+        ports = []
+        for p in self.image['ports']:
+            if p.get('expose', True):
+
+                r = "{}/{}".format(p['value'], p.get('protocol', 'tcp'))
+
+                if 'service' in p:
+                    r += ":{}".format(p['service'])
+                    ports.append(r)
+                else:
+                    # attempt to supply a service name by looking up the socket number
+                    try:
+                        service = socket.getservbyport(p['value'], p.get('protocol','tcp'))
+                        r += ":{}".format(service)
+                        ports.append(r)
+
+                    except OSError: # py3
+                        pass
+                    except socket.error: # py2
```

Comment:
Having two `except` clauses is failing the  Codacy/PR Quality Review. In Python 2, we could write `except OSError, socket.error` but that is not valid in Python 3. The issue is two different exception names/classes are used in 2 vs 3.